protocol/test/temple-zaps.ts

```diff
@@ -421,9 +499,22 @@ describe("Temple Stax Core Zaps", async () => {
       const tokenAmount = "1";
 
       await templeZaps.connect(owner).setSupportedStables([FRAX, FEI], [true, true]);
-      const pairContract = IERC20__factory.connect(TEMPLE_FRAX_PAIR, alice);
+      const pairContract = IUniswapV2Pair__factory.connect(TEMPLE_FRAX_PAIR, alice);
```

Comment:
I'm getting errors from running this locally. `balanceOf does not exist on type IUniswapV2Pair`. 
`pairContract` was init only to call the `balanceOf` function to check lp balances for test assertions.